models/aeif_cond_exp.h

```diff
@@ -107,6 +107,13 @@ For implementation details see the
 
 See also [1]_.
 
+.. admonition::
```

Comment:
```suggestion
.. note::

```

Reply:
The text does not render because the generic admonition directive requires a title; you can also use note. Also added a new line under the directive.